### ISSUE TITLE
Why are the UST Event Instance & UST Event Registration objects missing record types?

### ISSUE THREAD

**Issue by jacebryan** [SFDO-Community/Summit-Events-App#13]
I've noticed that there are 2 page layouts in the package for UST Event Instance and UST Event Registration but no record types. Is this on purpose? The default layout is set to the UDAR Event Instance Layout & UDAR Event Registration Layout (respectively).

**Comment by tcdahlberg** [comment]
The UST_Event__c object has record types that on the Web front end inform views throughout the form process. So as the forms are made I can make code choices on the record types of the event umbrella object.

The other objects in the scheme I personally never put record types on. They may have been added by my admins for business functionality on campus for page views. I just nabbed them when I lifted and shifted the app. 

I think part of the process of making this a more broadly useful app will be to address how audiences  and record types are defined. It may be as simple as record typing the event more broadly and emptying out the audience. After install the user could define these things as they make sense for their use.
